src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
+
+  stylesArray.forEach((style) => {
+    if (style.trim()) {
+      const [property, value] = style.split(':');
+
+      if (property && value) {
+        stylesObject[property.trim()] = value.trim();
+      }
+    }
+  });
```

Comment:
[CODE STYLE] - When chaining methods, it's best to have them at the same indentation level to enhance readability. Consider refactoring the code to chain `split` and `forEach` with proper indentation.